packages/messenger-widget/src/components/ConfigureProfile/chain/genome/ConfigureGenomeProfile.tsx

```diff
@@ -46,11 +46,22 @@ export const ConfigureGenomeProfile = () => {
         }
     };
 
-    const label = 'gNosis ';
+    const label =
```

Comment:
Thanks for adding a proper text. I had this at the back of my mind but somehow forgot to add it